src/main/java/thrift/commons/core/Messages.java

```diff
@@ -11,5 +11,6 @@
     public static final String MESSAGE_INVALID_TRANSACTION_DISPLAYED_INDEX =
             "The transaction index provided is invalid";
     public static final String MESSAGE_TRANSACTIONS_LISTED_OVERVIEW = "%1$d transactions listed!";
+    public static final String MESSAGE_INVALID_MONTH_FORMAT = "Invalid month format! \n%1$s";
```

Comment:
You should declare this in `ListCommandParser` class since it is related to that class

Reply:
removed from messages and put to LCP.